js/console.ts

```diff
@@ -108,6 +108,11 @@ function stringifyArgs(args: any[]): string {
   return out.join(" ");
 }
 
+// tslint:disable-next-line:no-any
+export function testStringify(...args: any[]): string {
+  return stringifyArgs(args);
+}
```

Comment:
Move this to console_test? Why not call stringifyArgs directly?

Reply:
I'd rather you just add the brackets than add this function.

Reply:
Oh sorry, I didn't see the move to `console_test` part. Yeah I'll make that change.